cmd/run.go

```diff
@@ -40,3 +55,87 @@ func (h *Handler) Run(ctx context.Context, req *entity.CommandRequest) error {
 
 	return nil
 }
+
+func (h *Handler) runInDocker(ctx context.Context, pwd string, envs *entity.Envs) error {
+	// Start building the image
+	projectCfg, err := h.cfg.GetProjectConfigs()
+	if err != nil {
+		return err
+	}
+
+	project, err := h.ctrl.GetProject(ctx, projectCfg.Project)
+	if err != nil {
+		return err
+	}
+
+	// Strip characters not allowed in Docker image names
+	sanitiser := regexp.MustCompile(`[^A-Za-z0-9_-]`)
+	imageName := sanitiser.ReplaceAllString(project.Name, "")
+	image := fmt.Sprintf("railway-local/%s:latest", imageName)
```

Comment:
Any reason for Railway-local vs just Railway? Our internal representation for image routes is Railway/ so I could see it either way (nice to differentiate but also maybe nice to unify?)

Reply:
Mostly just to differentiate. imo it makes it a bit clearer that this isn't the image that's actually running up there in Railway, but it's very specifically one built locally.

Reply:
Yup, I can totally vibe with this